app/services/paypal_service/api/lookup.rb

```diff
@@ -21,15 +21,15 @@ def with_account(cid, pid, &block)
       end
     end
 
-    def with_accounts(cid, pid, &block)
+    def with_accounts(cid, pid, receiver_id, &block)
       admin_acc = AccountStore.get(community_id: cid)
```

Comment:
That's right :) And a very good example about my comment above about the implicit uniqueness rules :)